metadata/reader.go

```diff
@@ -53,7 +53,13 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-// reader stores filesystem metadata parsed from ztoc to metadata DB
+const (
+	// bboltInsertionChunkSize roughly determines the maximum number of insertions to
+	// bbolt per transaction.
+	bboltInsertionChunkSize = 5000
```

Comment:
Can we document why 5000? Even a line stating "manually tuned and gave the best performance" could give context in a few years time.